modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ModalKeyDetails/index.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+import ClayAlert from '@clayui/alert';
+import ClayModal from '@clayui/modal';
+import React, {useState} from 'react';
+import Button from '../../../../common/components/Button';
+import TableKeyDetails from '../TableKeyDetails';
+
+const ModalKeyDetails = ({
+	ACTIVATION_STATUS,
```

Comment:
Uppercase pattern name must be used only for constants values, not for props, functions, params or variables when your value change it.

```suggestion
	activationStatus,
```